docker-compose.override.yml

```diff
@@ -5,3 +5,6 @@ services:
   app:
     ports:
       - "3000:3000"
+  db:
```

Comment:
do we need to expose this port?

Reply:
No, but it makes it easier to set up a development environment if you can deploy the DB with Docker and run the rest locally.

If you run everything in Docker locally, you probably want to be able to connect to the database to look around.

I'm not sure what will happen if you have PostgreSQL installed outside of Docker though. Maybe I should change the port to make sure they don't conflict?

Reply:
Actually, the reason I set the port this way is so that the local Cobra can connect to the one in Docker. Otherwise you need to set it manually. Maybe I should find a neater way to support either a local PostgreSQL or one in Docker?

Reply:
I'll change it to one that won't conflict with an installation on the host. It's useful to be able to connect to it during development.

Reply:
Oh! * Facepalm *

Sorry, I should read more carefully. This is great. Apologies for reviewing while dashing between things.